package.json

```diff
@@ -44,7 +44,7 @@
       },
       "brave-core": {
         "dir": "src/brave",
-        "branch": "master",
+        "branch": "ie_speedreader",
```

Comment:
will revert once CI is done

Reply:
cc @mihaiplesa 

we shouldn't need to do this in brave-browser, I believe CI for every PR branch creates a corresponding branch on b-b pointing to the right branch on b-c?

Reply:
yes, but there is something else in this PR

Reply:
CI puts the b-c branch automatically if it exists, just has to match the name of the b-b one (it won't push the change). You can just include your changes in `ie_speedreader` and not change `package.json`.